modules/gdscript/gdscript_editor.cpp

```diff
@@ -2745,8 +2745,6 @@ static void _find_call_arguments(GDScriptParser::CompletionContext &p_context, c
 	const GDScriptParser::CallNode *call = static_cast<const GDScriptParser::CallNode *>(p_call);
 	GDScriptParser::Node::Type callee_type = call->get_callee_type();
 
-	GDScriptCompletionIdentifier connect_base;
```

Comment:
Have this been discussed elsewhere? Otherwise this deserves investigating *why* it is unused

Reply:
Haven't discussed it elsewhere. It's a leftover from 3.x. Code completion for `connect` used to be done there but it is now handled in another function overload.